recipes/repo.rb

```diff
@@ -173,12 +238,15 @@
   only_if { ::File.symlink?("/home/#{agent_username}/.gnupg/S.gpg-agent") }
 end
 
-# Set up ssh authorized keys for publish over ssh.
-directory "/home/#{agent_username}/.ssh" do
```

Comment:
This is diff artifacting. There is no relationship to the added cookbook_file. This section was moved up to line 115 to allow the ssh directory to be used for GPG private keys.